editoast/src/models/infra.rs

```diff
@@ -323,4 +331,51 @@ pub mod tests {
         })
         .await;
     }
+
+    #[rstest]
+    async fn clone_infra_returns_new_cloned_infra() {
+        // GIVEN
+        let pg_db_pool = get_default_pg_db_pool();
+        let small_infra = &small_infra(db_pool()).await.model;
+        let infra_new_name = "cloned_infra_1".to_string();
```

Comment:
We're trying to isolate editoast tests as much as we can since they're sharing the same database instance. We decided to prefix objects names by the name of the test that uses them.

```suggestion
        let infra_new_name = "clone_infra_returns_new_cloned_infra __cloned_infra_1".to_string();
```

Reply:
Done.